pages/blog/new-asyncapi-tools-page.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'New Tools Dashboard for AsyncAPI'
+date: 2023-01-30T06:00:00+01:00
+type: Communication
+tags:
+  - Automation
+  - JSON Schema
+cover: /img/posts/new-asyncapi-tools-page/cover.webp
+authors:
+  - name: Akshat Nema
+    photo: /img/avatars/akshatnema.webp
+    link: https://twitter.com/AksNema
+    byline: AsyncAPI Maintainer and Dev Akshat Nema
+excerpt: "We haven't come up with some exciting features in AsyncAPI in recent, but now we come up with something new and special"
+featured: true
+---
+
+We already know that the present list of tools used inside AsyncAPI is presented on [AsyncAPI Tools Overview](/docs/tools) and it is manually maintained inside the Github repository. All the tools are sorted according to the different categories in which they are used and maintained by the contributors. But we don't have, how to filter the tools according to our interest 🤔, like languages, technolgies used in it, open source tools, etc. Also, what if you want to add your tool in the list, what you will do 🤔? Will make a PR?
+
+What if I say, you don't need to make a mess of PR now 😉. Yepp, we introduce you with something new to add tools in our website. For this, stay tuned in the blog 👇.
+
+# AsyncAPI Tool File
+
+We have introduced a new concept of `.asyncapi-tool` file which describes the type and details of a tool related to AsyncAPI. This file will follow certain schema and fields to describe your tool appropriately according to the needs of a user and it will then automatically being added to our website within a week. But the question comes, where this file will exist? inside AsyncAPI repositories? Definitely not!. Here comes the twist, this file will be created and maintained in your Tool's repository, and it won't ask for our approval. This file will follow certain schema which you have to follow to successfully and appropriately add your tool to our brand new [Tools Dashboard](/tools). The proper schema of the file is described [here](https://github.com/asyncapi/website/blob/master/scripts/tools/tools-schema.json). You can use the tools like [Online JSON Validator](https://www.liquid-technologies.com/online-json-schema-validator) to validate your JSON data for Tool against the schema given above. For more detailed explanation, you can refer to our [official documentation](https://github.com/asyncapi/community/blob/master/new-tool-documentation.md) of `.asyncapi-tool` file. 
```

Comment:
```suggestion
We have introduced a new concept of the `.asyncapi-tool` file which describes the type and details of a tool related to AsyncAPI. This file will follow certain schema and fields to describe your tool appropriately according to the needs of a user and it will then automatically be added to our website within a week. But the question comes, where this file will exist? Inside AsyncAPI repositories? Definitely not! Here comes the twist, this file will be created and maintained in your Tool's repository, and it won't ask for our approval. This file will follow a certain schema that you have to follow to successfully and appropriately add your tool to our brand new [AsyncAPI Tools Dashboard](/tools). The [proper schema of the file is described here](https://github.com/asyncapi/website/blob/master/scripts/tools/tools-schema.json). You can use the tools like the [Online JSON Validator](https://www.liquid-technologies.com/online-json-schema-validator) to validate your JSON data for Tools against the schema given above. For a more detailed explanation, you can refer to our [official `.asyncapi-tool` file documentation](https://github.com/asyncapi/community/blob/master/new-tool-documentation.md). 
```

Improving links for accessibility, so that screenreader users can tell where the links take them.